odoo/models.py

```diff
@@ -3970,7 +3970,7 @@ def create(self, vals_list):
         The new records are initialized using the values from the list of dicts
         ``vals_list``, and if necessary those from :meth:`~.default_get`.
 
-        :param list vals_list:
+        :param Union[list, dict] vals_list:
```

Comment:
The right type is actually a **`list` of `dict`** or a `dict`.

Reply:
@rco-odoo updated!